src/main/resources/AntwoordenModel_v2_2_2.csv

```diff
@@ -86,8 +86,8 @@ Ja,Ja,Nee,Ja,Ja,Ja,Een_ouder,,,,Ja,Ja,,,,Ja_ouder2,,21ao2,V,Nee,Nee,Nee,Ja,2171,
 Ja,Ja,Nee,Nee,,Ja,Een_ouder,,,,Ja,Ja,,,,Ja_beiden,,22,G,Nee,Nee,Nee,Nee,2250,Ingezeten - minderjarig - categorie 11 en recentere gebeurtenis(sen) - heeft één ouder - ouder gehuwd/partnerschap - ouder en partner onbevoegd - soort gezag is G - tijdelijk geen gezag
 Ja,Ja,Nee,Ja,Ja,Ja,Een_ouder,,,,Ja,Ja,,,,Ja_beiden,,22a,G,Nee,Nee,Nee,Nee,2251,Ingezeten - minderjarig - adoptie NL akte - categorie 11 en recentere gebeurtenis(sen) - heeft één ouder - ouder gehuwd/partnerschap - ouder en partner onbevoegd - soort gezag is G - tijdelijk geen gezag
 Ja,Ja,Nee,Nee,,Ja,Twee_ouders,,,,,,,,,,,23e,N,Nee,Nee,Nee,Nee,2300,Ingezeten - minderjarig - categorie 11 en recentere gebeurtenis(sen) - heeft twee ouders - exceptie - gezag is niet te bepalen omdat relevante gegevens ontbreken bij de bepaling van huwelijk/partnerschap. Het gaat om de volgende gegevens: gevens xxxxx
```

Comment:
@CathyDingemanse  @Patrick-4488  is er een reden dat deze toelichting niet is aangepast?